modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/MonadErrorContinuations.kt

```diff
@@ -1,15 +1,24 @@
 package arrow.typeclasses
 
+import arrow.Kind
+import arrow.typeclasses.suspended.MonadThrowSyntax
 import kotlin.coroutines.CoroutineContext
 import kotlin.coroutines.EmptyCoroutineContext
 import kotlin.coroutines.RestrictsSuspension
 
 @RestrictsSuspension
 @Suppress("DELEGATED_MEMBER_HIDES_SUPERTYPE_OVERRIDE")
 open class MonadErrorContinuation<F, A>(val ME: MonadError<F, Throwable>, override val context: CoroutineContext = EmptyCoroutineContext) :
```

Comment:
Yes but since we have ambitions to introduce `BIO` and I did not restructure continuation dependencies in this PR I figured they are all gonna be parametric to E when we have E in the Fx hierarchy. Feel free to jump in and rename at will.